### DISCUSSION TITLE
html report item "keyword search"

### DISCUSSION THREAD

**Discussion by gisms** [sepinf-inc/IPED#1445] [Q&A] [answered]
I didn't understand the functionality of the "keyword search" option in the html report.
I tried importing a list of keywords when generating the report, but it made no difference.

**Comment by lfcnassif** [comment, marked as answer]
It simply imports the keywords found in the search combo box in the analysis UI, try to click on the arrow on the right of the search box to see the the imported (and found) keywords

**Reply by gisms** [reply]
thank you!!
